[EDU-3086] feat: add secure overview (#473)
aziontech/docs: src/content/docs/en/pages/secure-journey/overview/overview.mdx

---
title: Secure Overview
description: Secure your applications and domains with edge computing protection.
meta_tags: secure, overview, firewall
namespace: docs_secure_overview
menu_namespace: secureMenu
permalink: /documentation/products/secure/overview/
---

import Button from '~/components/Button.astro'

Azion Secure products assist you in protecting your domains, applications, and infrastructure.

> *Secure* is a discipline that works along with the [build process](/en/documentation/products/guides/build/overview/) and integrates itself with observability. When you're building applications, security must be a part of the process from the beginning: they walk hand in hand with one another. Once the building and securing process is configured, you must also constantly [observe](/en/documentation/products/observe/) your applications and implemented security rules to guarantee they're working as expected and that you can make fast decisions and changes to configurations to continue securing them.

Reviewer: discipline pareceu esquisito na primeira leitura, mas não tenho certeza de qual outro termo poderia ir

> *Secure* is a discipline that works along with the [build process](/en/documentation/products/guides/build/overview/) and integrates itself with observability. When you're building applications, security must be a part of the process from the beginning: they walk hand in hand with one another. Once the building and securing process is configured, you must also constantly [observe](/en/documentation/products/observe/) your applications and implemented security rules to guarantee they're working as expected and that you can make fast decisions and changes to configurations to continue securing them.

Reviewer: ```suggestion
*Secure* is a discipline that works along with the [building process](/en/documentation/products/guides/build/overview/) and integrates itself with observability. When you're building applications, security must be a part of the process from the beginning: they walk hand in hand with one another. Once the building and securing process is configured, you must also constantly [observe](/en/documentation/products/observe/) your applications and implemented security rules to guarantee they're working as expected and that you can make fast decisions and changes to configurations to continue securing them.
```

> 
> ## Edge Firewall

Reviewer: ```suggestion

---

## Edge Firewall
```

---

## Edge Firewall

**Edge Firewall** is a programmable security product that allows you to secure your applications, your domains, and your user's domains.

<Button href="/en/documentation/products/edge-firewall/" variant="secondary" text="go to edge firewall documentation"></Button>

<Button href="/en/documentation/products/guides/secure/configure-main-settings/" variant="secondary" text="go to edge firewall main settings guide"></Button>
 
When a user attempts to access and attack your applications hosted on Azion with any type of attack, Edge Firewall and its modules act as a protection layer before the attack reaches your application and, consequently, the origin.

Edge Firewall is the core to protect your applications and domains but security rules and policies can be enhanced with [modules](/en/documentation/products/guides/secure/configure-main-settings/#modules): **DDoS Protection Unmetered**, **Edge Functions**, **Network Layer Protection**, and **Web Application Firewall**. As each of them has specific secure purposes, it's recommended to use all of them so your applications are protected with the entire stack.

> An attack attempt will pass through each of Edge Firewall's module and they'll block the attack according to their purposes and configured rules.

Reviewer 2: ```suggestion
An attack attempt will pass through each of Edge Firewall's modules and they'll block the attack according to their purposes and configured rules.
```

![Overview of a request encountering Edge Firewall](/assets/docs/images/uploads/edge-firewall-overview.png)

Furthermore, your configurations need to be set as rules on [Rules Engine](/en/documentation/products/edge-firewall/#rules-engine-for-edge-firewall) for them to be implemented and for requests to be properly analyzed, allowed, or blocked.

<Button href="/en/documentation/products/guides/secure/work-with-rules-engine/" variant="secondary" text="go to rules engine for edge firewall guide"></Button>

---

## Intelligent DNS

**Intelligent DNS** allows you to host your domains on Azion's distributed infrastructure, providing high performance, availability, and security.

<Button href="/en/documentation/products/intelligent-dns/" variant="secondary" text="go to edge firewall documentation"></Button>

<Button href="" variant="secondary" text="go to intelligent dns main settings guide"></Button>

> By adding and configuring [records](/en/documentation/products/guides/secure/add-records/), your domain's information are stored securely at Azion's infrastructure, and allow you to configure other products to properly handle requests, such as edge applications and edge firewalls.

Reviewer 2: ```suggestion
By adding and configuring [records](/en/documentation/products/guides/secure/add-records/), your domain's information is stored securely at Azion's infrastructure and allows you to configure other products to properly handle requests, such as edge applications and edge firewalls.
```
